.github/actions/action-utils.js

```diff
@@ -51,4 +51,7 @@ module.exports.friendlyDateFromISODate = function(isoDate) {
 
 module.exports.splitVersionTag = splitVersionTag;
 module.exports.readFile = (fileName) => util.promisify(fs.readFile)(fileName, 'utf8');
-module.exports.writeFile = (fileName, contents) => util.promisify(fs.writeFile)(fileName, contents);
\ No newline at end of file
+module.exports.readFileSync = (fileName) => fs.readFileSync(fileName, 'utf8');
```

Comment:
Sync variants of existing methods shouldn't be added here, the only reason we're exporting `readFile` here is because we need to promisify the existing `fs.readFile` and this is a common thing for all actions. `{readFile,writeFile}Sync` aren't adding anything when the stock `fs` version works